Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -37,6 +39,16 @@ public abstract class APIGatewayProxyFunction
         public bool EnableResponseLogging { get; set; }
 
         /// <summary>
+        /// If true then all response content will be Base64-encoded before being returned.
+        /// </summary>
+        public bool EnableBinaryResponseContent { get; set; }
+
+        /// <summary>
+        /// An optional set of Content-Types that will trigger encoding the response content in Base64.
+        /// </summary>
+        public IEnumerable<string> BinaryResponseContentTypes { get; set; }
```

Comment:
Was thinking about it a bit more -- does it make sense to support it "both ways"?  For example, you can establish a default behavior (to encode or not to encode), and then define a list of *exceptions* -- the content types that alter the default behavior.  And then of course, the default configuration is as you described which would accommodate most users.  Something like:

```csharp
enum ResponseEncoding
{
  String = 0, // Or UTF8 or something...
  Base64 = 1,
}

class APIGatewayProxyFunction
{
  protected Dictionary<string, ResponseEncoding> _responseEncodingForContentType =
      new Dictionary<string, ResponseEncoding>
      {
        // The defaults, such as from https://en.wikipedia.org/wiki/Media_type#Common_examples
        ["text/plain"] = ResponseEncoding.String,
        ["text/html"] = ResponseEncoding.String,
        ["text/css"] = ResponseEncoding.String,
        ["application/xml"] = ResponseEncoding.String,
        ["application/json"] = ResponseEncoding.String,
        // etc...
      };

  public ResponseEncoding DefaultResponseEncoding
  { get; set; } = ResponseEncoding.Base64;

  public void AddResponseEncodingForContentType(string contentType, ResponseEncoding encoding)
  {
    // ...
  }
}
```

Don't want to *explode* this feature change, but just thinking about the possible usage scenarios and what would be most stable for the long-term.